daggerverse/gcp-secrets/main.go

```diff
@@ -0,0 +1,146 @@
+// A generated module for GcpSecrets functions
+
+package main
+
+import (
+	"context"
+	"dagger/gcp-secrets/internal/dagger"
+	"errors"
+	"strings"
+
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	"cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+	"google.golang.org/api/option"
+)
+
+type GcpSecrets struct{}
+
+// Get a single secret from the Google Secret Manager with latest version
+// example: dagger call get-secret --project-id=project-id --secret-name=secret-name --version=latest --credential=path-to-Servcie-account-file
+func (m *GcpSecrets) GetSecretWithVersion(projectID string, secretName string, version string, credential *dagger.File) (string, error) {
```

Comment:
`version` could be defaulted to latest. 

https://docs.dagger.io/manuals/developer/functions/#default-values